packages/data/tests/unit/adapters/facts/elide-test.ts

```diff
@@ -524,4 +524,53 @@ module('Unit | Adapter | facts/elide', function(hooks) {
       assert.deepEqual(responseText, errors[0].messages, 'fetchDataForRequest an array of response objects on error');
     }
   });
+
+  test('escaped filter values', async function(assert) {
+    const adapter: ElideFactsAdapter = this.owner.lookup('adapter:facts/elide');
+    const EscapedTest: RequestV2 = {
+      table: 'table1',
+      columns: [
+        { field: 'table1.m1', type: 'metric', parameters: {} },
+        { field: 'table1.m2', type: 'metric', parameters: {} },
+        { field: 'table1.r', type: 'metric', parameters: { p: '123', as: 'a' } },
+        { field: 'table1.d1', type: 'dimension', parameters: {} },
+        { field: 'table1.d2', type: 'dimension', parameters: {} }
```

Comment:
```suggestion
```
Do we need all these example columns?

Reply:
No, I was just going off the other test request, I'll delete them